Instruments/Cessena_Citation_CJ4/CJ4-Ambient_Light_Dimmer/d6f6d4ab-62f6-4427-844a-4909fe2fbd4e/logic.lua

```diff
@@ -49,6 +50,13 @@ var_ambient_darkness = si_variable_create("sivar_ambient_darkness", "FLOAT", 0.0
 img_buttons = img_add_fullscreen("buttons.png")
 img_buttons_night = img_add_fullscreen("buttons_night.png")
 
+--Message Port for Hue Dimming
+currentambientdarkness = 0
+function callback_message_port_hue(message_port_hue1, payload)
```

Comment:
seems like this should be "callback_message_port_hue1", not ...hue with no number 1